File: docs/extensibility/codesnippet/CSharp/walkthrough-displaying-quickinfo-tooltips_12.cs
    internal TestQuickInfoController(ITextView textView, IList<ITextBuffer> subjectBuffers, TestQuickInfoControllerProvider provider)
    {
        m_textView = textView;
        m_subjectBuffers = subjectBuffers;
        m_provider = provider;

        m_textView.MouseHover += this.OnTextViewMouseHover;
    }

File: docs-old-1104/extensibility/codesnippet/VisualBasic/walkthrough-displaying-quickinfo-tooltips_12.vb
    Friend Sub New(ByVal textView As ITextView, ByVal subjectBuffers As IList(Of ITextBuffer), ByVal provider As TestQuickInfoControllerProvider)
        m_textView = textView
        m_subjectBuffers = subjectBuffers
        m_provider = provider

        AddHandler m_textView.MouseHover, AddressOf Me.OnTextViewMouseHover
    End Sub